diff -Naur harfbuzz-2.6.4/CMakeLists.txt harfbuzz-2.6.4.tpg/CMakeLists.txt
--- harfbuzz-2.6.4/CMakeLists.txt	2019-09-03 00:32:11.000000000 +0000
+++ harfbuzz-2.6.4.tpg/CMakeLists.txt	2019-12-17 12:11:51.506304910 +0000
@@ -209,7 +209,8 @@
   list(APPEND THIRD_PARTY_LIBS ${FREETYPE_LIBRARIES})
   include_directories(AFTER ${FREETYPE_INCLUDE_DIRS})
   add_definitions(-DHAVE_FREETYPE=1)
-
+  target_link_libraries(${PROJECT_SOURCE_DIR}/src/hb-ft.h ${FREETYPE_LIBRARIES})
+  target_include_directories(${PROJECT_SOURCE_DIR}/src/hb-ft.h ${FREETYPE_INCLUDE_DIRS})
   list(APPEND project_headers ${PROJECT_SOURCE_DIR}/src/hb-ft.h)
 
   # So check_funcs can find its headers
