diff -uraN harfbuzz-2.7.1/meson.build omv-harfbuzz-2.7.1/meson.build
--- harfbuzz-2.7.1/meson.build	2020-08-13 18:24:53.000000000 +0200
+++ omv-harfbuzz-2.7.1/meson.build	2020-08-16 00:58:31.269499276 +0200
@@ -95,7 +95,7 @@
                       fallback: ['glib', 'libglib_dep'])
 gobject_dep = dependency('gobject-2.0', required: get_option('gobject'),
                          fallback: ['glib', 'libgobject_dep'])
-fontconfig_dep = dependency('fontconfig', required: get_option('fontconfig'),
+fontconfig_dep = dependency('fontconfig1', required: get_option('fontconfig'),
                             fallback: ['fontconfig', 'fontconfig_dep'])
 graphite2_dep = dependency('graphite2', required: get_option('graphite'))
 
